docs/aineko_dream.md

```diff
@@ -0,0 +1,26 @@
+# Aineko Dream
+
+Aineko Dream is the quickest way to get started with Aineko pipelines by leveraging the power of generative AI to create a starter Aineko pipeline based on your use case. 
+
+## Generating a Project
+
+To generate a project, invoke the Aineko Dream CLI with a prompt with:
+
+```bash
+poetry run aineko dream "create a pipeline that scrapes twitter and analyses the results to identify trends" "API-KEY"
+```
+
+replacing `API-KEY` with a valid Aineko Dream API key. Contact support@aineko.dev to get an API key to try this feature.
```

Comment:
Linking to Slack here also might be good. A general response time is also helpful (ie expect a response within 72 hours)